mautic/mautic
# Import selecting matched column multiple - bug fix

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Issues addressed (#s or URLs) | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When importing contacts you have to match the "colums" of your csv-file with the corresponding fields in mautic. Currently it is possible to select the same colum several times and therefore importing the same value out of you csv-file to different fields in mautic. 
This will help to prevent selecting same column several times.

#### Steps to test this PR:
1. Apply PR 
2. Navigate to contacts section
3. Click import 
4. upload valid csv file to import
5. select matched column in contact field 
6. check whether the already selected column is listed in other drop down in contact field

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Navigate to contacts section
2. Click import 
3. upload valid csv file to import
4. select same column in all contact field
5. it is possible to select the same matched column in all contact field
